README.md

```diff
@@ -61,8 +61,8 @@ yarn playground
 
 Then, open your browser on the indicated URL and test the search bar 🙂
 
-The data comes from MeiliSearch documentation.<br>
-Type `create an indxe` to live the MeiliSearch experience with the [typo tolerance](https://docs.meilisearch.com/reference/under_the_hood/typotolerance.html).
+The data comes from Meilisearch documentation.<br>
+Type `create an indxe` to live the Meilisearch experience with the [typo tolerance](https://docs.meilisearch.com/reference/under_the_hood/typotolerance.html).
```

Comment:
```suggestion
Type `create an index` to live the Meilisearch experience with the [typo tolerance](https://docs.meilisearch.com/reference/under_the_hood/typotolerance.html).
```

Reply:
Its on purpose! To show the typetolerancy :)